### PR TITLE
Auth into liveblocksSessions

### PR DESCRIPTION
**Pull request by IvanRomero03**

### CODE REVIEW

File: src/app/api/liveblocks-auth/route.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        name: ctx.session?.user.name,
+      },
+    }, // Optional
+    // { userInfo: user.metadata }, // Optional
+  );
+
+  // Use a naming pattern to allow access to rooms with wildcards
+  // Giving the user read access on their org, and write access on their group
+  //   session.allow(`${user.organization}:*`, session.READ_ACCESS);
+  //   session.allow(`${user.organization}:${user.group}:*`, session.FULL_ACCESS);
```
Comment (Maxy673) [inline src/app/api/liveblocks-auth/route.ts]: Delete unnecessary line code comments if they not change about access

**Comment by Maxy673** [comment]
Add comments about what they changed  in general

**Review by Maxy673 — CHANGES_REQUESTED** [review]
changes only with comments on the lines of code

**Review by EstebanAleman21 — APPROVED** [review]
LGTM

**Review by Maxy673 — APPROVED** [review]
LGTM

**Pull request merged**